package.json

```diff
@@ -32,10 +32,10 @@
     "@babel/preset-env": "^7.8.7",
     "@babel/preset-react": "^7.8.3",
     "@svgr/rollup": "^4.3.0",
-    "babel-plugin-styled-components": "^1.10.0",
```

Comment:
I think this needs to go back in.

Reply:
I've removed the counterpart from `.bablerc` and everything still seems to build correctly, so I guess it was never needed... or has been superseded by something else!?

Reply:
If you read the description for what it does: https://github.com/styled-components/babel-plugin-styled-components

It sounds like something that we want.

Reply:
Yes, fair enough. I'll revert this...

Reply:
Fixed